proto/api/bria.proto

```diff
@@ -186,6 +187,20 @@ message FindAddressByExternalIdResponse {
   WalletAddress address = 2;
 }
 
+message GetAddressRequest {
+  oneof identifier {
+    string address = 1;
+    string external_id = 2;
+  }
+}
+
+message GetAddressResponse{
```

Comment:
missing `bool change_address`  and address is optional (when its change its null)